vkconfig/CHANGELOG.md

```diff
@@ -107,7 +107,7 @@
 - Add build date in the title bar
 
 ### Improvements:
-- Add link to [Vulkan Guide layers](https://github.com/KhronosGroup/Vulkan-Guide/blob/master/chapters/development_tools.md#vulkan-layers) list within the help menu
+- Add link to [Vulkan Guide layers](https://github.com/KhronosGroup/Vulkan-Guide/blob/main/chapters/development_tools.md#vulkan-layers) list within the help menu
```

Comment:
Also using `main` consistently helps with minor issues like this in the documentation.